### PR TITLE
Calc bmi with use another file

### PR DESCRIPTION
**Pull request by Kukuruza1998**

### CODE REVIEW

File: calc_bmi_read_write_another_file/main.py [hunk 1]
```diff
@@ -0,0 +1,206 @@
+import os
+
+def print_menu():
+  file = open('calc_bmi_read_write_another_file/menu.txt', encoding='utf-8')
```
Comment (dvk-net) [inline calc_bmi_read_write_another_file/main.py:4]: utf-8 is set by default

File: calc_bmi_read_write_another_file/main.py [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+  for i in file:
+    print(i.strip())
+  file.close()  
+
+def clear_screen():
+  os.system('CLS')
+
+def processing_user_choice_menu():
+  a = input("Ваш выбор: ")
```
Comment (dvk-net) [inline calc_bmi_read_write_another_file/main.py:13]: what is a?
It's always better to give descriptive names

File: calc_bmi_read_write_another_file/main.py [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+  if a.isdigit():
+    a = int(a)
+    if 1 <= a <= 6:
+      return a
+  clear_screen()
+  print('Некорректный выбор!',end="\n"*2)
```
Comment (dvk-net) [inline calc_bmi_read_write_another_file/main.py:19]: where is no way to reenter..

File: calc_bmi_read_write_another_file/main.py [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+
+def start_func_for_menu_item(a):
+  if a==1:
```
Comment (dvk-net) [inline calc_bmi_read_write_another_file/main.py:22]: spaces (PEP8)
if a == 1

**Review by dvk-net — COMMENTED** [review]
11 open vs 8 close - hm...
I would use **with open**
and json or pickle

Good

File: calc_bmi_read_write_another_file/main.py (hunk 1 above)
Comment (Kukuruza1998) [reply calc_bmi_read_write_another_file/main.py:4]: image

Comment (Kukuruza1998) [reply calc_bmi_read_write_another_file/main.py:4]: but when specify a value utf-8, run without error

File: calc_bmi_read_write_another_file/main.py (hunk 4 above)
Comment (Kukuruza1998) [reply calc_bmi_read_write_another_file/main.py:22]: my bad

**Pull request merged**